worker/upgradedatabase/package_test.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package upgradedatabase
+
+import (
+	"testing"
+
+	gc "gopkg.in/check.v1"
+)
+
+//go:generate go run go.uber.org/mock/mockgen -package upgradedatabase -destination worker_test.go github.com/juju/juju/worker/upgradedatabase Logger
```

Comment:
Generate this into package_mock_test.go, and the other one into lock_mock_test.go.

We'll be using worker_test.go for actual tests.